templates/default/modprobe.conf.erb

```diff
@@ -1,6 +1,23 @@
 # This file was generated by Chef
-<% @config.each do |cmd, values| -%>
-<%   values.each do |v| -%>
-<%= cmd %> <%= @name %> <%= v %>
-<%   end -%>
-<% end -%>
+<% if @blacklist %>
+<%= 'blacklist ' + @name if @blacklist %>
+<% else -%>
+<%   case @alias
+     when String -%>
+<%= 'alias ' + [@alias, @name].join(' ') %>
+<%   when Array  -%>
+<%= @alias.sort.map { |v| "options #{v} #{@name}" }.join("\n") %>
+<%   end if @options-%>
```

Comment:
should be `@alias`, but I would event prefer that you don't check that it's truthy.